modules/abstract-lightning/src/wallet/lightning.ts

```diff
@@ -94,6 +98,37 @@ export interface ILightningWallet {
    * @returns {Promise<unknown>} A promise resolving to the updated wallet response or throwing an error if the update fails.
    */
   updateWalletCoinSpecific(params: UpdateLightningWalletClientRequest): Promise<unknown>;
+
+  /**
+   * Get transaction details by ID
+   * @param {string} txId - Transaction ID to lookup
+   * @returns {Promise<Transaction>} Transaction details
+   * @throws {InvalidTxId} When transaction ID is not valid
+   */
+  getTransaction(txId: string): Promise<Transaction>;
+
+  /**
+   * List transactions for a wallet with optional filtering
+   * @param {TransactionQuery} params - Query parameters for filtering transactions
+   * @returns {Promise<Transaction[]>} List of transactions
```

Comment:
Give details for these query params - like [here](https://github.com/BitGo/BitGoJS/pull/5675/files#diff-a2d3b25a72da16ff6284bf4efe89295adfb93fefb4f95874eb1694f38b8d4350R50-R60).

Same for other APIs as well.